### PR TITLE
mezzanine: 3dtof: updated guides and sdk link

### PR DESCRIPTION
**Pull request by ric96**
Also change product name to "Analog Device’s 3D Time Of Flight Mezzanine Stack"

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: mezzanine/3dtof/README.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
-title: Documentation for Analog Devices 3DToF Mezzanine
+title: Documentation for Analog Device’s 3D Time Of Flight Mezzanine Stack
```
Comment (sgermanserrano) [inline mezzanine/3dtof/README.md]:
```suggestion
title: Documentation for Analog Devices 3D Time Of Flight Mezzanine Stack
```

File: mezzanine/3dtof/README.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 permalink: /documentation/mezzanine/3dtof/
 ---
-# 3D Imaging Time of Flight Development Platform
+# Analog Device’s 3D Time Of Flight Mezzanine Stack
```
Comment (sgermanserrano) [inline mezzanine/3dtof/README.md]:
```suggestion
# Analog Devices 3D Time Of Flight Mezzanine Stack
```

File: mezzanine/3dtof/README.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 
 - 1) [Introduction](#introduction)
-	- 1.1) [About the Analog Devices 3DToF Mezzanine](#11-about-the-analog-devices-3dtof-mezzanine)
+	- 1.1) [About the Analog Device’s 3D Time Of Flight Mezzanine Stack](#11-about-the-analog-devices-3d-time-of-flight-mezzanine-stack)
```
Comment (sgermanserrano) [inline mezzanine/3dtof/README.md]:
```suggestion
	- 1.1) [About the Analog Devices 3D Time Of Flight Mezzanine Stack](#11-about-the-analog-devices-3d-time-of-flight-mezzanine-stack)
```

File: mezzanine/3dtof/README.md [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
 	- 1.2) [Features & Specifications](#12-features-and-specifications)
 - 2) [Guides](#2-guides)
 - 3) [Files](#3-files)
 
 ## 1) Introduction
-### 1.1) About the Analog Devices 3D Imaging Time of Flight Development Platform
+### 1.1) About the Analog Device’s 3D Time Of Flight Mezzanine Stack
```
Comment (sgermanserrano) [inline mezzanine/3dtof/README.md]:
```suggestion
### 1.1) About the Analog Devices 3D Time Of Flight Mezzanine Stack
```

File: mezzanine/3dtof/guides/README.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 ---
-title: Shiratech LTE Guides
+title: Analog Device’s 3D Time Of Flight Mezzanine Stack Guides
```
Comment (sgermanserrano) [inline mezzanine/3dtof/guides/README.md]:
```suggestion
title: Analog Devices 3D Time Of Flight Mezzanine Stack Guides
```

Comment (ric96) [reply mezzanine/3dtof/guides/README.md]: I have to update this on the website as well but ADI specifically requested for:
"Analog Device’s 3D Time Of Flight Mezzanine Stack"

Comment (sgermanserrano) [reply mezzanine/3dtof/guides/README.md]: OK, disregard comments then :)

**Pull request merged**